org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/ClosureTestCase.xtend

```diff
@@ -32,7 +22,7 @@ class ClosureTestCase extends AbstractWollokInterpreterTestCase {
 		'''
 		program p {
 			var to = "world"
-			val helloWorld = ["hello " + to ]
+			val helloWorld = {->"hello " + to }
```

Comment:
But the funny part is that the not arrow syntax was nearly not used in the whole wollok code base! I only fixed 2 or 3 occurrences I think. 

With the exception of Scala - which have partial application - and Ruby - well, when you don't use the arrow syntax -, I think having the no args arrow is pretty common.